src/components/workflow/Workflow.vue

```diff
@@ -29,17 +26,25 @@
    import SelectFiles from '@/components/workflow/SelectFiles.vue';
    import SelectWorkflow from '@/components/workflow/SelectWorkflow.vue';
    import Loader from '@/components/shared/Loader.vue';
+   import Logout from '@/components/shared/Logout.vue'
    
    export default {
      name: 'Workflow',
      components:{
        SelectFiles,
        SelectWorkflow,
        Sidebar,
-       Loader
+       Loader,
+       Logout
      },
     computed:{
       workflowSubmission: sync('workflowSubmission')
     }
    }
-</script>
\ No newline at end of file
+</script>
+<style lang="css">

```

Comment:
Should this be scoped to this page only?

Reply:
yeah. we could. Will add that.